mod-orders/mod-orders.postman_collection.json

```diff
@@ -811,7 +811,8 @@
 									"script": {
 										"id": "fa42b974-2ac7-4dbf-9572-3f42ef3569f6",
 										"exec": [
-											"pm.environment.set(\"identifierTypeName\", \"Vendor Title Number\");"
+											"pm.environment.set(\"identifierTypeName\", \"Vendor Title Number\");",
+											"pm.environment.set(\"productIdTypeId\", \"8261054f-be78-422d-bd51-4ed9f33c3422\");"
```

Comment:
The id should not be hard-coded. This request should be rewritten to create new identifier type specific for API Tests and use its uuid in further requests